tools/ci_build/windows/extract_nuget_files.ps1

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This file is used by Zip-Nuget Packaging NoContribOps Pipeline,Zip-Nuget-Java Packaging Pipeline
+
+# Re-construct a build directory that contains binaries from all the different platforms we're including
+# in the native ORT nuget package
+$nuget_artifacts_dir = "$Env:BUILD_BINARIESDIRECTORY\RelWithDebInfo\RelWithDebInfo\nuget-artifacts"
```

Comment:
Is it possible to separate things out a little better so steps like this don't have a dependency on a build output dir from a local build?

i.e. copy ALL the required build artifacts to one place ($Env:BUILD_BINARIESDIRECTORY\build-artifacts) and extract them as needed to setup the directory structure the nuget pack needs (e.g. somewhere like $Env:BUILD_BINARIESDIRECTORY\nuget would be fine). 

That way in theory anything could use the build artifacts as they're all in one place and we cleanly separate out the nuget specific usage.

Reply:
unpack to ($Env:BUILD_BINARIESDIRECTORY\nuget-ext)